README.md

```diff
@@ -7,15 +7,16 @@ Install and manage Determinate Nix.
 ```nix
 {
   inputs.nix.url = "https://flakehub.com/f/DeterminateSystems/nix/2.0";
+  inputs.nixpkgs.url = "https://flakehub.com/f/DeterminateSystems/nixpkgs/0.2405.*";
```

Comment:
We don't publish nixpkgs to `DeterminateSystems/nixpkgs`

Reply:
Derp